scripts/populate_conferences.py

```diff
@@ -505,6 +505,27 @@ def main():
         'poster': True,
         'talk': True,
     },
+    'psichi2015': {
+        'name': 'PsiChi',
+        'info_url': 'psichi.org',
```

Comment:
This should be `http://www.psichi.org`

In the absence of a protocol (http://), the "Conference homepage" link at the top of the page points to:
`http://localhost:5000/view/psichi2015/psichi.org`